tests/instrumentation/test_bootstrap.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from opentelemetry.instrumentation import (
+    bootstrap as otel_bootstrap,
+)
+
+from elasticotel.instrumentation import bootstrap
+
+
+def test_overriden_instrumentations(monkeypatch, capfd):
```

Comment:
```suggestion
def test_edot_priority_instrumentations(monkeypatch, capfd):
```

Feel free to reject the code suggestion. I think this is indeed overriding, but maybe it is more about priority since the code isn't declaring "opentelemetry-instrumentation-openai-v2" in its requirements? I feel like override would be to switch from opentelemetry-instrumentation-openai-v2 -> elastic-opentelemetry-instrumentation-openai and priority means to choose instead.

Reply:
I don't think it's about priority since we are removing `opentelemetry-instrumentation-openai-v2` from the packages the tool knows about and adding ours instead.

Reply:
thanks I misunderstood what was going on